core/atomic_c11.c

```diff
@@ -137,12 +137,12 @@ typedef uint64_t I8;
 #define TEMPLATE_ATOMIC_FETCH_OP_N(opname, op, n, prefixop) \
     I##n __atomic_fetch_##opname##_##n(volatile void *ptr, I##n val, int memmodel) \
     { \
-        unsigned int mask = irq_disable();    \
-        (void)memmodel;                       \
-        I##n tmp = *(I##n *)ptr;              \
-        *(I##n *)ptr = prefixop(tmp op val);  \
-        irq_restore(mask);                    \
-        return tmp;                           \
+        unsigned int mask = irq_disable();             \
+        (void)memmodel;                                \
+        I##n tmp = *(volatile I##n *)ptr;              \
+        *(volatile I##n *)ptr = prefixop(tmp op val);  \
```

Comment:
weird, github still shows this line's last backslash one off, but with a direct checkout, it is alright?

Reply:
On my browser (Firefox) they look all aligned.

I also changed the column of the backslashes as requested to be  `4 * n`, and not `4 * n + 1`. So merge?

Reply:
Looks good in github now. You might F5 on this, Kaspar^^

Reply:
I F5'ed the shit out of this, still shows the missing space. Anyhow, it also doesn't let me upload the screenshot. Anyways, directly checking this out looks fine, so let's go!